net.sourceforge.uhexen2.uhexen2.yml

```diff
@@ -0,0 +1,61 @@
+app-id: net.sourceforge.uhexen2.uhexen2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: glhexen2
+
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/GL
+    version: '23.08'
+
+finish-args:
+  # hardware 3D
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+  # Data path
+  - --persist=.hexen2
+
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  # Add this to compile with SDL2 support instead
+  # - shared-modules/SDL2/SDL-2-with-libdecor.json
+  - shared-modules/libmad/libmad.json
+
+  - name: uhexen2
+    sources:
+      - type: git
+        url: https://github.com/sezero/uhexen2
+        tag: uhexen2-1.5.9
+        commit: 4ef664bc41e3998b0d2a55ff1166dadf34c936be
+        x-checker-data:
+          type: git
+          tag-pattern: ^uhexen2-(.*)$
+      - type: file
+        path: net.sourceforge.uhexen2.uhexen2.desktop
+      - type: file
+        path: net.sourceforge.uhexen2.uhexen2.appdata.xml
+    buildsystem: simple
+    build-commands:
+      # Uncomment this to apply the SDL2 patch
+      # - git apply patches/sdl2.patch
+      - cd engine/hexen2 && make glh2 && make -C server
+      - cd engine/hexenworld && make -C client glhw && make -C server
+      - cd hw_utils/hwmaster && make
+      # Install the executables we've built
+      - install -D -m755 -t /app/bin/ ./engine/hexen2/glhexen2 ./engine/hexen2/server/h2ded ./engine/hexenworld/client/glhwcl ./engine/hexenworld/server/hwsv hw_utils/hwmaster/hwmaster
+      # Install metadata and launcher files
+      - install -D -m755 ./engine/resource/hexen2n.png /app/share/icons/hicolor/128x128/apps/net.sourceforge.uhexen2.uhexen2.png 
```

Comment:
this icon is 70x70. It neither big enough nor a standard size.
128x128 is the minimum, or a SVG.

That's why the build fail.